receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -1498,8 +1498,22 @@ func TestTargetInfoWithMultipleRequests(t *testing.T) {
 		},
 	}
 
+	expectedIndex0 := func() pmetric.Metrics {
+		metrics := pmetric.NewMetrics()
+		rm := metrics.ResourceMetrics().AppendEmpty()
+		attrs := rm.Resource().Attributes()
+		attrs.PutStr("service.namespace", "production")
+		attrs.PutStr("service.name", "service_a")
+		attrs.PutStr("service.instance.id", "host1")
+		attrs.PutStr("machine_type", "n1-standard-1")
+		attrs.PutStr("cloud_provider", "gcp")
+		attrs.PutStr("region", "us-central1")
+
+		return metrics
+	}()
```

Comment:
Hmmm, interesting! That means we're passing through a ResourceMetrics object without any metrics...

Should we even do that? I'm not sure what's expected from a receiver, are we fine if we pass this to the next consumer?

Reply:
> Should we even do that? I'm not sure what's expected from a receiver, are we fine if we pass this to the next consumer?

Reading the docs, I didn't find anything... Do you know someone that could help?